LDEventSource/LDDataAccumulator.m

```diff
@@ -0,0 +1,31 @@
+//
+//  LDDataAccumulator.m
+//  DarklyEventSource
+//
+//  Created by Mark Pokorny on 5/30/18. +JMJ
+//  Copyright © 2018 Catamorphic Co. All rights reserved.
+//
+
+#import "LDDataAccumulator.h"
+
+@implementation LDDataAccumulator
```

Comment:
DataAccumulator is a fairly general name, but `isReadyToParseEvent` clearly implies this is not a general use class, but a narrow class intended to accumulate a string for use in parsing an event.

I wonder if we can clear this up by either:

a. Giving this a more specific name - `LDEventStringAccumulator`
b. Keeping this generic by removing `isReadyToParseEvent` and having it be the responsibility of another class (category on NSString? class function on `LDEventParser`?) (In this case, we may still want to call this class `LDStringAccumulator`, but also, it does almost nothing at this point)

Reply:
Renamed to LDEventStringAccumulator